instrumentation/docker/src/main/scala/net/cakesolutions/testkit/logging/docker/DockerLogSource.scala

```diff
@@ -61,10 +59,16 @@ object DockerLogSource extends LoggingSource[Json] {
       if (exit != 0 && exit != 143) {
         throw ProcessTerminated(exit) // FIXME: really?
       }
-      if (! cancelP.isCompleted) {
+      if (!cancelP.isCompleted) {
         cancelP.success(())
         subscriber.onComplete()
       }
     }
   }
 }
+
+object DockerLogSource {
+
+  final case class ProcessTerminated(exitCode: Int)
```

Comment:
Should this be in `core` module rather than in the `instrumentation/docker` module?